Source/IntegrationTests/IntegrationTests.csproj

```diff
@@ -21,8 +21,7 @@
     </ItemGroup>
 
     <ItemGroup>
-        <ProjectReference Include="..\DafnyDriver\DafnyDriver.csproj" />
-        <ProjectReference Include="..\DafnyRuntime\DafnyRuntime.csproj" />
```

Comment:
Based on the test output, it seems that this project still depends dynamically on `DafnyRuntime`.

Reply:
Specific tests are depending on linking to `DafnyRuntime.dll`, yup. Unfortunately one of them reveals that `DafnyMain` calls `Assembly.LoadFile` on any `dll` file passed to the CLI, in order to read the contents of any `DafnySourceAttribute`: https://github.com/dafny-lang/dafny/blob/master/Source/Dafny/DafnyMain.cs#L67. This will fail if its dependencies are not available (`DafnyRuntime.dll` or any other dependency!).

I originally fixed this by loading the assembly reflectively using Cecil, but unfortunately we lost that when switching to .NET Core: https://github.com/dafny-lang/dafny/pull/794/files#diff-0ba2e4842e187b0efcf6a6b5fd6fca73b546ece27e54074265eab734f826240dL45-R47. Even better, there's an `Assembly.ReflectionOnlyLoad` method, but it turns out it always did a regular load instead, and now always throws an exception instead (which is definitely better, but still feels like a bait-and-switch :P )

Reply:
Eventually figured out the supported replacement for `Assembly.ReflectionOnlyLoad` - see updated PR description and release note.